scripts/vue_or_react.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# cookiecutter tn-spa-bootstrapper --config-file tn-spa-bootstrapper/cookiecutter/vue_template.yaml --no-input
+git fetch origin main
+ts_count=$(git diff --name-only origin/main -- | grep "\.ts$" | wc -l)
+tsx_count=$(git diff --name-only origin/main -- | grep "\.tsx$" | wc -l)
+react_count=$(($ts_count + $tsx_count))
+vue_count=$(git diff --name-only origin/main -- | grep "\.vue$" | wc -l)
+
+if [ $react_count -gt $vue_count ]; then
+  echo "cookiecutter/react_template.yaml"
+else
+  echo "cookiecutter/vue_template.yaml"
+fi
```

Comment:
Script to determine Vue or React.
Basically we look at the list of files changed when comparing this branch to main.
We then count the number of files based on extensions. (ts/tsx vs vue)
The larger count wins.
Default for 1st build is React

Reply:
this may start to fail when we move Vue over to TS (although vue wont use tsx) why not target the specific client folder?